server/src/helpers/game/commands/gm/GMCreateGold.ts

```diff
@@ -0,0 +1,23 @@
+import { IMacroCommandArgs, IPlayer } from '../../../../interfaces';
+import { MacroCommand } from '../../../../models/macro';
+
+export class GMCreateGold extends MacroCommand {
+
+  aliases = ['@gold'];
+  isGMCommand = true;
+  canBeInstant = false;
+  canBeFast = false;
+
+  execute(player: IPlayer, args: IMacroCommandArgs) {
+    const gold = this.game.itemCreator.getSimpleItem('Gold Coin');
+    const value = +args.arrayArgs[0];
+    if (!value) {
+      this.sendMessage(player, `Enter an amount of gold.`);
+      return;
+    } else {
```

Comment:
The `else` is superfluous if you do `if() { return }` - basically pull this code out of the else block.

Reply:
Thanks, done